tests/test_enforcer.py

```diff
@@ -21,9 +13,18 @@ def __init__(self, name, age):
         self.name = name
         self.age = age
 
-class TestConfig(TestCase):
+class TestCaseBase(TestCase):
+    def get_enforcer(self, model=None, adapter=None, enable_log=False):
+        return casbin.Enforcer(
+            model,
+            adapter,
+            enable_log,
+        )
+
```

Comment:
Unnecessarily use `get_enforcer` as a class member method.

Reply:
I made SyncedEnforcer a drop-in replacement for Enforcer. Therefore i wanted to run all tests for `Enforcer` and `SyncedEnforcer`. To derive all tests i made `get_enforcer` a method, which is then overwritten in `TestConfigSynced`.

If you have a better solution in mind, which does not require to duplicate all tests, please let me know.